test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
Sure, but now how do we know that -h still works? Looks like there is no longer a -h test.